workflows/genome_annotation/annotation-braker3/Genome_annotation_with_braker3.ga

```diff
@@ -0,0 +1,659 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "This workflow uses Braker3 to annotate a genome.",
```

Comment:
Could you extend this a little ? Maybe the first sentence from the readme ?]
```suggestion
    "annotation": "This workflow allows for genome annotation using Braker3 and evaluates the quality of the annotation with BUSCO and genome annotation statistics.",
```

Could you make the busco step optional ? I think there are different preferences wrt what to use to assess genome completeness but especially for large genomes we found busco to be a bottleneck.